public/API/API_GetUserRankAndScore.php

```diff
@@ -25,8 +25,8 @@
 $softcorePoints = 0;
 
 if ($user) {
-    $points = $user?->points ?? 0;
-    $softcorePoints = $user?->points_softcore ?? 0;
```

Comment:
I didn't realize we already had helpers/aliases for these. I can see how that would make the conversion more complicated. Thanks for doing it.